transactions.go

```diff
@@ -408,16 +410,6 @@ func (lw *LibWallet) countTransactions(query storm.Query) (int, error) {
 	return count, nil
 }
 
-func (lw *LibWallet) FilterTransactions(txFilter int32) (string, error) {
-	query := lw.parseTxFilter(txFilter)
-	return lw.queryTransactions(query)
-}
-
-func (lw *LibWallet) CountTransactions(txFilter int32) (int, error) {
-	query := lw.parseTxFilter(txFilter)
-	return lw.countTransactions(query)
-}
-
 func (lw *LibWallet) DetermineTxFilter(txHash []byte) (int32, error) {
```

Comment:
The properties needed to determine a tx filter are `tx.Type` and `tx.Direction`. From the use case in decred/dcrandroid#378, those information already exist in the context, no need to pass a txHash and reconstruct the tx just to get those info. You can modify this function to take `tx.Type` and `tx.Direction` or probably just do the determination of tx filter on dcrandroid directly.